swagger-parser/index.d.ts

```diff
@@ -0,0 +1,10 @@
+// Type definitions for swagger-parser 4.x
+// Project: https://www.npmjs.com/package/swagger-parser
+// Definitions by: Tobias Wolff <https://github.com/Tobias4872/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace SwaggerParser {
+  function parse(path: string): any;
+}
+
+export = SwaggerParser;
```

Comment:
Could just be `export function parse(path: string): any;`.
There also appear to be [options](https://github.com/BigstickCarpet/swagger-parser/blob/releases/4.0.0/docs/swagger-parser.md#parseapi-options-callback) for this function.
Also, this could return `Promise<any>`.